lib/membrane_aac_fdk_plugin/encoder.ex

```diff
@@ -83,6 +83,14 @@ defmodule Membrane.AAC.FDK.Encoder do
                 type: :integer,
                 spec: pos_integer() | nil,
                 default: nil
+              ],
+              input_stream_format: [
+                spec: RawAudio.t(),
+                type: :stream_format,
+                default: nil,
+                description: """
+                Input type - used to set input sample rate and channels.
+                """
```

Comment:
`def_options` is used to define options passed in spec, eg. `child(:source, %Source{option: value})`. There is no need to put `input_stream_format` here, because it won't be passed in spec, it is just another field in `state`